seagence/manifest.json

```
@@ -0,0 +1,70 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "94f4e504-c98c-466f-b934-5e5ee0331944",
+  "app_id": "seagence",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Defect Monitoring and Root Cause Automation using Seagence",
+    "title": "Seagence",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Defect and Success Clusters",
+        "image_url": "images/defect-and-success-clusters.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Success and Defect Execution Paths",
+        "image_url": "images/defect-and-successexecution-paths-1440x810.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Seagence Defects Overview dashboard",
+        "image_url": "images/datadog-dashboard.png.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Supported OS::HP-UX",
+      "Supported OS::Solaris",
+      "Supported OS::IBM i/OS",
+      "Supported OS::IBM z/OS",
+      "Supported OS::iOS",
+      "Submitted Data Type::Events",
+	  "Category::Alerting",
+	  "Category::Automation",
+      "Category::Event Management",
+	  "Offering::Integration"
+    ]
+  },
+  "assets": {
+	"integration": {
+      "source_type_name": "Seagence",
+      "configuration": {},
+      "events": {
+        "creates_events": true
+      },
+	  "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+	},
+	"dashboards": {
+      "seagence_overview": "assets/dashboards/seagence_overview.json"
+    },
+    "monitors": {"Seagence Defect Detection": "assets/monitors/defect_detection_monitor.json"},
+    "logs": {}
```

If you're not doing anything with logs, you can just remove this field.
```suggestion
    "monitors": {"Seagence Defect Detection": "assets/monitors/defect_detection_monitor.json"}
```